support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -115,4 +115,23 @@ export const tests: Tests = {
 		seed: 10,
 		optimizeId: 'dQCXBc3QQIW7M1Di_qSCHw',
 	},
+	singleToRecurring: {
+		variants: [
```

Comment:
I believe based on the spec for this AB test there should be 3 cohorts: `control`, `variantA` and `variantB`? The copy inside the container will differ `variantA` and `variantB`.

Reply:
Yes, implemented and redescribed correctly on PR